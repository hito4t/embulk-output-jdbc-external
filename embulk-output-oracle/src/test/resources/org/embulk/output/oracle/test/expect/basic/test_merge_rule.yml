table: TEST_MERGE1
mode: merge
merge_rule: ["VARCHAR2_ITEM = 'x'", "NUMBER_ITEM = T.NUMBER_ITEM + S.NUMBER_ITEM"]
