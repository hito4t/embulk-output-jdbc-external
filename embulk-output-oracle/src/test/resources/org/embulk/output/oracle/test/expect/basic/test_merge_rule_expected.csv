A001,aaa,99.99
A002,x,111.1
A003,ccc,77.77
A004,x,
A005,,
A006,FFF,66.66
